src/services/database.js

```diff
@@ -4,19 +4,27 @@ import fs from 'fs';
 
 const database = { connection: null };
 
+export function getFilePathList(paths) {
+  return (
+    fs
+      .readdirSync(path.join.apply(null, paths))
+      // only takes data files
+      .filter(filename => filename.endsWith('.csv'))
+      // prepend path to filename
+      .map(filename => path.join.apply(null, [...paths, filename]))
+  );
+}
+
 const filenames = [
-  '../database.csv',
-  ...fs.readdirSync(path.join(__dirname, '../../data')),
+  path.join(__dirname, '../../database.csv'),
+  ...getFilePathList([__dirname, '../../data']),
```

Comment:
according to the new function introduced, you should ever 

1. pass a single string as parameter and not an array
2. or keep the previous paradigm : generate an array of folder first, then pass it to you function

Reply:
Indeed, corrected  as well in the new PR.